pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1520,8 +1520,11 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       // No need to restart if nothing exists
       return Promise.resolve({});
     }
+    const reasons: RestartReasons = {};
 
-    return Promise.resolve(this.kubeBackend.requiresRestartReasons(this.cfg, cfg));
+    Object.assign(reasons, this.kubeBackend.requiresRestartReasons(this.cfg, cfg, { 'experimental.rdNetworking': undefined }));
```

Comment:
We only ever call `Object.assign(reasons, …)` once; that means there's no point in doing that when we could just
```suggestion
    return this.kubeBackend.requiresRestartReasons(this.cfg, cfg, { 'experimental.rdNetworking': undefined });
```
(Also, we might want to wrap this line.)

Reply:
That makes sense 👍 

> (Also, we might want to wrap this line.)

I did try the line wrap but linter hates me :|